code/modules/mob/mob_movement.dm

```diff
@@ -287,25 +287,23 @@
 				//drunk wheelchair driving
 				if(mob.confused && prob(20))
 					direct = pick(cardinal)
-				move_delay += 2
+				move_delay += 2 + config.vehicle_delay
 				return mob.buckled.relaymove(mob,direct)
 
-		var/mob/living/carbon/human/H
+		var/tally = mob.movement_delay() + config.walk_speed
 
+		// Apply huuman specific modifiers.
 		if (ishuman(mob))
-
-			H = mob
-			var/tally = mob.movement_delay() + config.walk_speed
+			var/mob/living/carbon/human/H = mob
 			//If we're sprinting and able to continue sprinting, then apply the sprint bonus ontop of this
-			if (H.m_intent == "run" && H.species.handle_sprint_cost(H, tally))//This will return false if we collapse from exhaustion
-				tally = tally/(1+H.sprint_speed_factor)

```

Comment:
I'm still more in favour of making these delay multipliers be speed multipliers instead, so a higher value is faster, and you'd divide instead of multiply here.